test/rekt/features/broker/feature.go

```diff
@@ -42,6 +44,159 @@ import (
 	"knative.dev/reconciler-test/pkg/resources/service"
 )
 
+func BrokerWithManyTriggers() *feature.Feature {
+	f := feature.NewFeatureNamed("broker With Many Triggers")
+
+	// Construct different type, source and extensions of events
+	any := eventingv1.TriggerAnyFilter
+	eventType1 := "type1"
+	eventType2 := "type2"
+	eventSource1 := "http://source1.com"
+	eventSource2 := "http://source2.com"
+	// Be careful with the length of extension name and values,
+	// we use extension name and value as a part of the name of resources like subscriber and trigger,
+	// the maximum characters allowed of resource name is 63
+	extensionName1 := "extname1"
+	extensionValue1 := "extval1"
+	extensionName2 := "extname2"
+	extensionValue2 := "extvalue2"
+	nonMatchingExtensionName := "nonmatchingextname"
+	nonMatchingExtensionValue := "nonmatchingextval"
+
+	eventFilters1 := make(map[string]eventTestCase)
+	eventFilters1["dumper-"+"1"] = neweventTestCase(any, any)
+	eventFilters1["dumper-"+"12"] = neweventTestCase(eventType1, any)
+	eventFilters1["dumper-"+"123"] = neweventTestCase(any, eventSource1)
+	eventFilters1["dumper-"+"1234"] = neweventTestCase(eventType1, eventSource1)
+
+	eventFilters2 := make(map[string]eventTestCase)
+	eventFilters2["dumper-"+"12345"] = neweventTestCaseWithExtensions(any, any, map[string]interface{}{extensionName1: extensionValue1})
+	eventFilters2["dumper-"+"123456"] = neweventTestCaseWithExtensions(any, any, map[string]interface{}{extensionName1: extensionValue1, extensionName2: extensionValue2})
+	eventFilters2["dumper-"+"1234567"] = neweventTestCaseWithExtensions(any, any, map[string]interface{}{extensionName2: extensionValue2})
+	eventFilters2["dumper-"+"654321"] = neweventTestCaseWithExtensions(eventType1, any, map[string]interface{}{extensionName1: extensionValue1})
+	eventFilters2["dumper-"+"54321"] = neweventTestCaseWithExtensions(any, any, map[string]interface{}{extensionName1: any})
+	eventFilters2["dumper-"+"4321"] = neweventTestCaseWithExtensions(any, eventSource1, map[string]interface{}{extensionName1: extensionValue1})
+	eventFilters2["dumper-"+"321"] = neweventTestCaseWithExtensions(any, eventSource1, map[string]interface{}{extensionName1: extensionValue1, extensionName2: extensionValue2})
+	eventFilters2["dumper-"+"21"] = neweventTestCaseWithExtensions(any, eventSource2, map[string]interface{}{extensionName1: extensionValue1, extensionName2: extensionValue1})
```

Comment:
What's the reason to use `"dumper-"+"1"` rather than `"dumper-1"`? same for other keys

Reply:
> What's the reason to use `"dumper-"+"1"` rather than `"dumper-1"`? same for other keys

ha, It's from dumper + ramdon string, dumper1 is better, I'll update them.